src/backend/utils/__mocks__/elastic.js

```diff
@@ -11,11 +11,10 @@ const deletePost = (postId) => {
   return Promise.resolve();
 };
 
-// TODO
 const search = () =>
   Promise.resolve({
-    results: 0,
-    values: [],
+    results: Object.keys(db).length,
+    values: db,
```

Comment:
`values` should look like:

```js
[
 { id, text, score },
  ...
]
```

Where `id` is the id of the document (postId), `text` is the text of the post`, and score is a number between 1 and 10.  We could always return 10.

Reply:
```js
  values: Object.entries(db).map((id, text) => { id, text, 10 })
```

Or do it in the `indexPost` above and make it look the way you want, then you can use `Object.values()`

Reply:
Changed the mock to be identical to live ES